### PR TITLE
HTMLEditor: Global editorStylingMode property should be applied to the component(T1144373)

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

**Review by Krijovnick — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

File: js/ui/html_editor/ui.html_editor.js [hunk 1]
```diff
@@ -62,7 +63,7 @@ const HtmlEditor = Editor.inherit({
 
             imageUpload: null,
 
-            stylingMode: 'outlined'
+            stylingMode: config().editorStylingMode || 'outlined',
```
Comment (ksercs) [inline js/ui/html_editor/ui.html_editor.js:66]: It's a good idea to use nullish coalescing in such cases

File: testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js [hunk 2]
```diff
@@ -121,4 +122,27 @@ export default function() {
...
+
+    QUnit.module('Props from global config', {
+        beforeEach() {
+            config({
+                editorStylingMode: 'filled',
+            });
+        }, 
+        afterEach() {
+            config({
+                editorStylingMode: 'outlined',
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js:134]: Seems this test is not isolated

File: testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js [hunk 3]
```diff
@@ -121,4 +122,27 @@ export default function() {
...
+            });
+        }
+    }, () => {
+
+        test('editorStylingMode should be applyed', function(assert) {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js:139]: "applied"

File: testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js [hunk 4]
```diff
@@ -121,4 +122,27 @@ export default function() {
...
+            const $element = $('#htmlEditor').dxHtmlEditor({
+                value: 'Test'
+            });
+
+            assert.strictEqual($element.hasClass(HTML_EDITOR_OUTLINED_CLASS), false, 'has no old styling mode class');
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js:144]: seems like it's enough to check an option value

**Pull request merged**